public-docs/how-to-extend-graphql-to-add-field.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How To: Extend GraphQL to add a field
+---
+
+Sometimes you only need to extend GraphQL to add a field to an existing type. Here's how to do it.
+
+## Extend the schema
+
+1. If it doesn't already exist, create `/server/no-meteor/schemas` folder in the plugin, and add an `index.js` file there.
+1. If it doesn't already exist, create `schema.graphql` in `/server/no-meteor/schemas` in the plugin.
+1. Import the GraphQL file into `index.js` and default export it in an array:
+
+    ```js
+    import schema from "./schema.graphql";
+
+    export default [schema];
+    ```
+
+    > NOTE: For large plugins, you can split to multiple `.graphql` files and export a multi-item array.
+
+1. In the `.graphql` file, use GraphQL language to extend the type with your custom field.
+
+    ```js
```

Comment:
You can use a `graphql` code block here, like

```
```graphql
```

Renders like:

<img width="861" alt="screen shot 2018-10-16 at 3 15 53 pm" src="https://user-images.githubusercontent.com/3673236/47050756-6663a880-d156-11e8-9fc0-77b8eec3a6c8.png">

Reply:
Done